src/components/UI/docs/Breadcrumbs.tsx

```diff
@@ -7,7 +7,8 @@ export const Breadcrumbs: FC = () => {
   const router = useRouter();
 
   let pathSplit = router.asPath.split('/');
```

Comment:
@corwintines Would suggest just adding a # split here
```suggestion
  let pathSplit = router.asPath.split('#')[0].split('/');
```

If there is no `#`, then it will just return the original string... if there is, we only look at anything before the first one when splitting by `/`